src/Catty/ViewController/Continue&New/MaintainObject/MaintainSounds/SoundsTableViewController+SelectFile.swift

```diff
@@ -0,0 +1,72 @@
+/**
+ *  Copyright (C) 2010-2021 The Catrobat Team
+ *  (http://developer.catrobat.org/credits)
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  An additional term exception under section 7 of the GNU Affero
+ *  General Public License, version 3, is available at
+ *  (http://developer.catrobat.org/license_additional_term)
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+import MobileCoreServices
+
+extension SoundsTableViewController {
+
+    @objc
+    func showSoundsSelectFile() {
+        var documentPicker: UIDocumentPickerViewController
+        if #available(iOS 14.0, *) {
+            documentPicker = UIDocumentPickerViewController.init(forOpeningContentTypes: [UTType.wav, UTType.mp3], asCopy: true)
```

Comment:
I would suggest creating constants at the beginning of this extension so that one can easily find where to add new formats (e.g., `supportedAudioFormats`).